pmml/utils/test_accuracy_utils.py

```
@@ -0,0 +1,31 @@
+        validate_accuracy_score(0.74)
+
+
+def test_validate_accuracy_score_rejects_non_finite_scores():
+    for value in (math.nan, math.inf, -math.inf):
+        with pytest.raises(ValueError, match='finite number'):
+            validate_accuracy_score(value)
+
+
+def test_validate_accuracy_score_rejects_scores_outside_unit_range():
+    for value in (-0.1, 1.1):
+        with pytest.raises(ValueError, match='between 0.0000 and 1.0000'):
```

Ruff (0.15.18)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)